lib/view.dart

```diff
@@ -531,14 +535,46 @@ html.SelectElement _getDropdown(String id, List<String> options,
   return dropdownSelect;
 }
 
-void renderSettingsTab() {
-  clearContentTab();
-  content.append(html.DivElement()..innerText = 'Settings');
+void renderSettingsTab(String config) {
+  var wrapper = html.DivElement();
+  content.append(wrapper);
+
+  var textArea = html.TextAreaElement()..text = config;
+  wrapper.append(textArea);
+
+  var editor = code_mirror.CodeMirror.fromTextArea(textArea,
+      options: {'mode': 'javascript', 'lineNumbers': true});
```

Comment:
does it have a json mode, or is the javascript mode able to suitably display json?

Reply:
Ah yes, javascript is able to display json as it is a subset, but now added explicit mode restricting it to just json